test/jdk/tools/jpackage/junit/jdk.jpackage/jdk/jpackage/internal/AppImageFileTest.java

```diff
@@ -21,6 +21,11 @@
  * questions.
  */
 
+/*
+ * @test
+ * @modules jdk.jpackage
```

Comment:
Hello Matthias,

Would moving `@modules jdk.jpackage` to `test/jdk/tools/jpackage/junit/TEST.properties` like the following:

```
modules=jdk.jpackage
```

and adding the `@test` declaration to these  tests, get this passing on AIX? That way it will avoid the repeated `@modules` declaration in these test files.

Reply:
Christian noted that the `@test` in each of the test files may not even be needed if you could just add the `modules=jdk.jpackage` to the existing content of `TEST.properties` file.